facebook/OpenBIC
# wc: mb: Support vr sensor reading

Summary:
- Support vr sensor Renesas-isl69259.
- Fix sensor reading problem in previous pr.

Test Plan:
- Build code: PASS
- Check sensor read from BMC console

Log:
- BMC console
  ```
  root@bmc-oob:~# sensor-util mb
  mb:
  MB MB Inlet Temp             (0x1) :   35.00 C     | (ok)
  MB MB Outlet Temp            (0x2) :   30.00 C     | (ok)
  MB PCH Temp                  (0x4) :   44.00 C     | (ok)
  MB CPU Temp                  (0x5) :   85.00 C     | (ucr)
  MB SOC Therm Margin          (0x6) :  -15.00 C     | (ok)
  MB CPU TjMax                 (0x7) :  100.00 C     | (ok)
  MB SSD0 Temp                 (0x8) :   29.00 C     | (ok)
  MB HSC Temp                  (0x9) :   38.00 C     | (ok)
  MB DIMMA Temp                (0xA) :   32.00 C     | (ok)
  MB DIMMC Temp                (0xB) :   31.00 C     | (ok)
  MB DIMME Temp                (0xC) :   32.00 C     | (ok)
  MB DIMMG Temp                (0xD) :   31.00 C     | (ok)
  MB VCCIN SPS Temp            (0xE) :   45.00 C     | (ok)
  MB FIVRA SPS Temp            (0xF) :   43.00 C     | (ok)
  MB EHV SPS Temp              (0x10) :   39.00 C     | (ok)
  MB VCCD SPS Temp             (0x11) :   40.00 C     | (ok)
  MB FAON SPS Temp             (0x12) :   42.00 C     | (ok)
  MB P12V_STBY Vol             (0x20) :   11.71 Volts | (ok)
  MB P3V_BAT Vol               (0x21) :    0.77 Volts | (ok)
  MB P3V3_STBY Vol             (0x22) :    3.28 Volts | (ok)
  MB P1V8_STBY Vol             (0x23) :    1.70 Volts | (ok)
  MB P1V05_PCH Vol             (0x24) :    1.00 Volts | (ok)
  MB P5V_STBY Vol              (0x25) :    4.77 Volts | (ok)
  MB P12V_DIMM Vol             (0x26) :   11.74 Volts | (ok)
  MB P1V2_STBY Vol             (0x27) :    1.20 Volts | (ok)
  MB P3V3_M2 Vol               (0x28) :    3.27 Volts | (ok)
  MB HSC Input Vol             (0x29) :   11.94 Volts | (ok)
  MB VCCIN VR Vol              (0x2A) :    1.77 Volts | (ok)
  MB FIVRA VR Vol              (0x2B) :    1.80 Volts | (ok)
  MB EHV VR Vol                (0x2C) :    1.80 Volts | (ok)
  MB VCCD VR Vol               (0x2D) :    1.14 Volts | (ok)
  MB FAON VR Vol               (0x2E) :    0.97 Volts | (ok)
  MB HSC Output Cur            (0x40) :    3.88 Amps  | (ok)
  MB VCCIN VR Cur              (0x41) :   37.70 Amps  | (ok)
  MB FIVRA VR Cur              (0x42) :    5.00 Amps  | (ok)
  MB EHV VR Cur                (0x43) :    0.80 Amps  | (ok)
  MB VCCD VR Cur               (0x44) :    1.70 Amps  | (ok)
  MB VCCD VR Cur in            (0x45) :    0.23 Amps  | (ok)
  MB FAON VR Cur               (0x46) :    9.20 Amps  | (ok)
  MB CPU PWR                   (0x60) :   86.00 Watts | (ok)
  MB HSC Input Pwr             (0x61) :   42.00 Watts | (ok)
  MB DIMMA PMIC_Pout           (0x62) :    0.00 Watts | (ok)
  MB DIMMC PMIC_Pout           (0x63) :    0.12 Watts | (ok)
  MB DIMME PMIC_Pout           (0x64) :    0.12 Watts | (ok)
  MB DIMMG PMIC_Pout           (0x65) :    0.12 Watts | (ok)
  MB VCCIN VR Pout             (0x66) :   66.00 Watts | (ok)
  MB FIVRA VR Pout             (0x67) :    8.00 Watts | (ok)
  MB EHV VR Pout               (0x68) :    1.00 Watts | (ok)
  MB VCCD VR Pout              (0x69) :    2.00 Watts | (ok)
  MB FAON VR Pout              (0x6A) :    8.00 Watts | (ok)
  ```
